body/setup.py

```diff
@@ -28,6 +28,7 @@
                       'terminado==0.8.3; python_version < "3.0"', # depend on old terminado for py2
                       'dynamixel-sdk>=3.1; python_version >= "3.0.0"', # py2 gets dynamixel-sdk through ROS
                       'pyyaml>=5.1', # required for yaml.FullLoader
-                      'hello-robot-stretch-tool-share>=0.2.0' # defines other Stretch end effectors
+                      'hello-robot-stretch-tool-share>=0.2.0', # defines other Stretch end effectors
+                      'hello-robot-stretch-factory>=0.2.0'
```

Comment:
If Stretch Body is upgraded, and the migration tool is run, the library itself will run fine, but it's tools will not. Upgrading Stretch Body doesn't upgrade the tools themselves. We'll have to add Stretch Body Tools >=0.2.0 as a dependency as well.

Reply:
I've added 'hello-robot-stretch-body-tools>=0.3.0' , however note that RE1_migrate_params.py is installed via 'hello-robot-stretch-factory>=0.2.0'